app/controllers/api/v1/users_controller.rb

```diff
@@ -44,11 +46,11 @@ def signup
       private
 
       def login_params
-        params.permit(:email, :password)
+        params.require(:email, :password)
       end
 
       def signup_params
-        params.permit(:name, :email, :password, :password_confirmation, :confirmed_at)

```

Comment:
You removed `:password_confirmation`, so We are not using it to register a user?

Reply:
@wandergithub because I thought that this logic it's more convenient to do in the front-end. Otherwise, it would have to be done in the backend (maybe adding something in devise config in the User model). I didn't mean to complicate it. I think we could discuss it later; for now I could ask Aleazar